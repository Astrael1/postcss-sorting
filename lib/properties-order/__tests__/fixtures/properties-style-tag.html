<!DOCTYPE html>
<html>
<head>
	<style>
		a {
			z-index: 2;
			top: 0;
			position: absolute;
			display: block;
		}
	</style>
</head>
<body>
</body>
</html>
