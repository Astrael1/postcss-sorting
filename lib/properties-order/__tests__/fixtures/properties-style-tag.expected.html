<!DOCTYPE html>
<html>
<head>
	<style>
		a {
			position: absolute;
			top: 0;
			display: block;
			z-index: 2;
		}
	</style>
</head>
<body>
</body>
</html>
